src/Connector.AzureDataLake/Connector/AzureDataLakeClient.cs

```diff
@@ -31,9 +37,8 @@ public async Task SaveData(AzureDataLakeConnectorJobData configuration, string c
                 HttpHeaders = new PathHttpHeaders { ContentType = "application/json" }
```

Comment:
Should we switch on the fileName's extension now that we support formats other than json

Reply:
that method is no longer used because we are using `OpenWriteAsync`. but yeah we should allow setting of type